source/simulator/lateral_averaging.cc

```diff
@@ -308,6 +308,42 @@ namespace aspect
         const GravityModel::Interface<dim> *gravity_model;
         std::vector<Tensor<1,dim> > velocity_values;
     };
+
+
+
+    template <int dim>
+    class FunctorDepthAverageFieldMass: public internal::FunctorBase<dim>
+    {
+      public:
+        FunctorDepthAverageFieldMass(const FEValuesExtractors::Scalar &field)
+          : field_(field)
```

Comment:
You would usually turn this around, call the stored member variable `field` and the function parameter `field_`. That way you can use the nicer variable name further down.

Reply:
This is consistent with the implementation of all the other functors.